dired-collapse.el

```diff
@@ -131,7 +133,8 @@ filename (for example when the final directory is empty)."
             ;; to them, while dired-collapse requires all the details. So we disable invisibility here
             ;; temporarily.
             (buffer-invisibility-spec nil)
-            (inhibit-read-only t))
+            (inhibit-read-only t)
+            (rgx (and dired-omit-mode (dired-omit-regexp))))
```

Comment:
@Boruch-Baum the var `dired-omit-mode` and the function `dired-omit-regexp` may not be defined here as they come from `dired-x`, which is not required. Either require it or check for the symbols are bound and true and an actual function.

Reply:
Thanks @wyuenho I updated both instances and pushed a fix.

Reply:
You need to guard `dired-omit-regexp` as well